VaticanBAV/et/BAVet131.xml

```diff
@@ -27,52 +27,1023 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        <locus from="10r" to="10v"/>
+                                        <title type="complete" xml:lang="gez" ref="LIT3186Meeraf#Wednesday"/>
+                                        <textLang mainLang="gez"/>
+                                        <note>The section includes Pss 63, 64.</note>
+                                    </msItem>         
+                                    <msItem xml:id="ms_i1.2.4">
+                                        <locus from="10v" to="11r"/>
+                                        <title type="complete" xml:lang="gez" ref="LIT3186Meeraf#Thursday"/>
+                                        <textLang mainLang="gez"/>
+                                        <note>The section includes Pss 81, 82.</note>
+                                    </msItem>         
+                                    <msItem xml:id="ms_i1.2.5">
+                                        <locus target="#11r"/>
+                                        <title type="complete" xml:lang="gez" ref="LIT3186Meeraf#Friday"/>
+                                        <textLang mainLang="gez"/>
+                                        <note>The section includes Pss 111, 112.</note>
+                                    </msItem>         
+                                    <msItem xml:id="ms_i1.2.6">
+                                        <locus from="11r" to="11v"/>
+                                        <title type="complete" xml:lang="gez" ref="LIT3186Meeraf#SaturdayM"/>
+                                        <textLang mainLang="gez"/>
+                                        <note>The section includes Pss 131, 132.</note>
+                                    </msItem>         
+                                    <msItem xml:id="ms_i1.2.7">
+                                        <locus target="#11v"/>
+                                        <title type="complete" xml:lang="gez" ref="LIT3186Meeraf#SaturdayE"/>
+                                        <textLang mainLang="gez"/>
+                                        <note>The section includes Pss 1, 2.</note>
+                                    </msItem>         
+                                    <msItem xml:id="ms_i1.2.8">
+                                        <locus from="11v" to="12r"/>
+                                        <title type="complete" xml:lang="gez" ref="LIT3186Meeraf#Sunday"/>
+                                        <textLang mainLang="gez"/>
+                                        <note>The section includes Pss 148-150.</note>
+                                    </msItem>         
+                                </msItem>
+                                <msItem xml:id="ms_i1.3">
+                                    <locus from="12r" to="27r"/>
+                                    <title type="complete" xml:lang="gez" ref="LIT3186Meeraf#Mawaddes">Third section, on the office for the night</title>
```

Comment:
Mawaddǝs is not a general "office of the night". I think it would be methodologically better to use the precise Ethiopian-Eritrean liturgical terms rather than inventing English approximations (in general, nothing about how you did it, Massimo :) )

Reply:
Not easy to untangle this particular constellation of texts, on which I confess I do not have much experience. I only translated "Sectio III, de officio nocturno" and added the ID based on the elements at disposal (order, incipits, and so). Are there reliable titles? I might simply leave `<title type="complete" xml:lang="gez" ref="LIT3186Meeraf#Mawaddes"/>` without further specification

Reply:
There are titles in Velat's edition (with variants from the manuscripts he used in the apparatus). I think leaving  <title type="complete" xml:lang="gez" ref="LIT3186Meeraf#Mawaddes"/> without further specification is a perfect solution